include/libmcx.h

```diff
@@ -12,20 +12,6 @@
 
 /* don't change these macros and structures which is referred in mcx code */ 
 
-#ifndef TRUE
```

Comment:
Ok so I *think* these TRUE/FALSE/ERROR are actually used for the return values of the functions in here.